plugins/reporters/cyclonedx/src/funTest/kotlin/CycloneDxReporterFunTest.kt

```diff
@@ -94,6 +96,30 @@ class CycloneDxReporterFunTest : WordSpec({
             }
         }
 
+        "create the expected XML file with authors" {
```

Comment:
I believe instead of adding a new separate test, it would make sense to instead enhance the input of `"create the expected XML file"`. For enhancing it `withAuthors()` imo is ok. Furthermore, the analog JSON test case IMO should also use / keep on using the same input, so that XML and JSON tests remain analog.

Reply:
@MarcelBochtler would you agree with above? (and update the commit)

Reply:
> @MarcelBochtler would you agree with above? (and update the commit)

I would agree if the tests would pass.

But as the test is currently expected to fail, it would either have to be disabled or not being merged at all.
If I add the `withAuthors()` to the `"create the expected XML file"` file, we can no longer disable the test.

I'm not sure how to continue here.

Reply:
> I'm not sure how to continue here.

I think the best we can do is to either push at https://github.com/CycloneDX/cyclonedx-core-java/issues/638 for a fix, or contribute the fix ourselves.

Reply:
> I would agree if the tests would pass.

If we re-target this commit to update the test input data to illustrate an issue with the authors (not serialized), wouldn't the be ok? (And once it's fixed upstream and integrated, the expected results would be updated for good). (If this was merged as-is, then we have to revert it and update the existing tests, which seems unnecessary work)